src/framework/application.js

```diff
@@ -1136,14 +1136,15 @@ pc.extend(pc, function () {
                 if (assets[i].type === "texture" && assets[i].resource) {
                     assets[i].resource.destroy();
                 }
-                assets[i].unload();
+                assets[i].unloadOld();

```

Comment:
OK, we probably just want to change it everywhere to be unload, no point having an extra function.